docker/docker-compose.yml

```diff
@@ -4,7 +4,7 @@ version: "3"
 services:
   relay_paseo_alice:
     container_name: alice
-    image: "parity/polkadot:v1.1.0"
+    image: "parity/polkadot:v1.13.0"
```

Comment:
Was this version label hard to find? To ensure that it stays in sync, we may want to make a note and add it to the upgrade wiki.

Reply:
Great suggestion. I'll add it to the wiki.